src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  let total = days * pricePerDay;
+
+  if (days >= 3 && days < 7) {
+    total -= smallDiscount;
+  } else if (days >= 7) {
+    total -= bigDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    total -= bigDiscount;
  } else if (days >= 3 ) {
    total -= smallDiscount;
  }
```

You can swap these statements by place and get rid of one redundant check